packages/picker/src/picker.css

```diff
@@ -35,6 +35,41 @@ governing permissions and limitations under the License.
     --spectrum-picker-width: var(--spectrum-global-dimension-size-2400);
 }
 
+:host([loading]) .sp-picker-loading {
+    margin-inline-start: var(
+        --mod-picker-spacing-text-to-alert-icon-inline-start,
+        var(--spectrum-picker-spacing-text-to-alert-icon-inline-start)
+    );
+}
+
+:host([loading]) .sp-picker-loading-s {
+    --spectrum-progress-circle-size: var(--spectrum-workflow-icon-size-75);
+    --spectrum-progress-circle-thickness: var(
+        --spectrum-progress-circle-thickness-small
+    );
+}
```

Comment:
Should I pick some other variables? Or make my own ones? More context: the picker has 4 dimensions: s/m/l/xl and the loading spinner has 3 dimensions: s/m/l, and do not match together. So I had to adapt its dimensions. Also, should we validate with design those dimensions?

Reply:
@pfulton is there any token support for these variation?

Reply:
From a selector standpoint, we can use `:host([size="s"]) sp-progress-circle` instead of needing to add a dynamic class to this element.